src/evented-tokenizer.ts

```diff
@@ -10,12 +13,12 @@ export default class EventedTokenizer {
   public line: number = -1;
   public column: number = -1;
 
-  constructor(private delegate, private entityParser) {
+  constructor(public delegate, private entityParser) {
```

Comment:
Why is the delegate public?

Reply:
I guess it doesn't need to be and just requires refactoring on the glimmer side.